.flake8

```diff
@@ -1,12 +1,6 @@
-[metadata]
-license_file = LICENSE
-
 [flake8]
 max-line-length = 100
```

Comment:
Why not move these to `pyproject.toml` as well?

Reply:
Because that configuration format is not supported by flake8 ([docs](https://flake8.pycqa.org/en/6.0.0/user/configuration.html)).

Reply:
I thought I migrated flake options one, but I need to find it again. Looking back at a few projects, they at least have it in `pre-commit-config.yaml`

Reply:
I'd recommend flake8 to be replaced with ruff, but at a later time (not this PR)